src/routes.json

```diff
@@ -148,18 +148,16 @@
         "title": "Ideas"
     },
     {
-        "name": "jobs",
+        "name": "jobs-redirect",
         "pattern": "^/jobs/?$",
```

Comment:
While we are updating this, one thing about this kind of pattern is that it won't match URLs generated when shared on social media or through email campaigns, which often add `?bunch_of_trackers=1l23kj12l3kj...` query parameters to URLs automatically. We started adding this for simple routes like these that get linked to frequently in email/social media. Could you add the pattern that you see e.g. at the end of the `/parents` route on line 193 to the end of here?

Reply:
✔️ Done